todo_app/test/test_view_model.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from todo_app.view_model import ViewModel
+from todo_app.Item import Item
+
+def test_todo_items():
+    items=[Item(1,'test','TODO',22)]
+    item_view_model=ViewModel(items)
+    item_view_model.todo_items
+    assert len(item_view_model.todo_items) ==1  
```

Comment:
You're not actually testing that any filtering is happening. It would be better to create a list with multiple items in it and then check that only the TODO ones are returned by `view_model.todo_items`.